core-ui/src/shared/ResourceForm/components/Label.scss

```diff
@@ -0,0 +1,3 @@
+.info-tooltip {
+  cursor: help;
+}
```

Comment:
we have a global style for this named `.has-tooltip`, so I think it could be deleted